pkg/session/bootstrap.go

```diff
@@ -3232,6 +3252,8 @@ func doDDLWorks(s sessiontypes.Session) {
 	mustExecute(s, CreateGlobalTaskHistory)
 	// Create tidb_import_jobs
 	mustExecute(s, CreateImportJobs)
+	// Create tidb_br_jobs
+	mustExecute(s, CreateBrJobs)
```

Comment:
You need to update the bootstrap version, maybe also implement the upgrade to functions.